osu.Game.Rulesets.Mania/UI/ColumnFlow.cs

```diff
@@ -92,10 +104,27 @@ private void onSkinChanged()
         /// <param name="content">The content.</param>
         public void SetContentForColumn(int column, TContent content) => columns[column].Child = content;
 
-        public new MarginPadding Padding
+        private void updateMobileSizing()
         {
-            get => base.Padding;
-            set => base.Padding = value;
+            if (!IsLoaded)
+                return;
+
+            for (int i = 0; i < stageDefinition.Columns; i++)
+            {
+                // GridContainer+CellContainer containing this stage (gets split up for dual stages).
+                Vector2 containingCell = this.FindClosestParent<Stage>().Parent!.DrawSize;
+
+                // Best effort until we have better mobile support.
+                if (RuntimeInfo.IsMobile)
+                {
+                    // These numbers are based on mobile phones, aspect ~1.92.
+                    float mobileAdjust = 2.83f * Math.Min(1, 7f / stageDefinition.Columns);
```

Comment:
Is `2.83f` anything? `1.92f` is explained, but not this.

Reply:
ah, this is taken from the ratio applied in the previous version of this code (`170 / 60`).